queue_services/account-mailer/src/account_mailer/worker.py

```diff
@@ -45,36 +46,42 @@
 FLASK_APP.config.from_object(APP_CONFIG)
 db.init_app(FLASK_APP)
 
-MESSAGE_TYPE = 'account.mailer'
-
 
 async def process_event(event_message: dict, flask_app):
-    """Render the org status."""
+    """Process the incoming queue event message."""
     if not flask_app:
         raise QueueException('Flask App not available.')
 
     with flask_app.app_context():
-        if event_message.get('type', None) == MESSAGE_TYPE:
+        message_type = event_message.get('type', None)
+        email_msg = None
+        email_dict = None
+        if message_type == 'account.mailer':
+            email_msg = json.loads(event_message.get('data'))
+            email_dict = payment_completed.process(email_msg)
+            
+        elif message_type == 'bc.registry.payment.refundRequest':
             email_msg = json.loads(event_message.get('data'))
-            logger.debug('Extracted email msg: %s', email_msg)
-            process_email(email_msg, FLASK_APP)
+            email_dict = refund_requested.process(email_msg)
 
+        logger.debug('Extracted email msg: %s', email_dict)
+        process_email(email_msg, FLASK_APP)
```

Comment:
this should be 'email_dict' instead of email_msg right?

Reply:
Yes, thank you.  Fixed.